StripePaymentSheet/StripePaymentSheet/Source/PaymentSheet/Saved Payment Method Screen/SavedPaymentOptionsViewController.swift

```diff
@@ -321,7 +321,9 @@ class SavedPaymentOptionsViewController: UIViewController {
 
     func didFinishPresenting() {
         DispatchQueue.main.asyncAfter(deadline: .now().advanced(by: .milliseconds(750))) {
-            self.updateFormElement()
+            if self.isViewLoaded {
+                self.updateFormElement()
+            }
```

Comment:
Would be good to have a comment for why we wait 750 milliseconds and then call this.